terraform/layers/base/s3.tf

```diff
@@ -0,0 +1,13 @@
+module "s3_bucket" {
```

Comment:
It would be better to choose a more descriptive name, think about if we decide to add another S3 Bucket at a later date - how would we differentiate between the two? Also if we wanted to change this name down the line it would cause terraform to destroy and recreate the resource, possibly causing a number of changes throughout the codebase

Reply:
Looking above a name like 's3_webfiles_bucket' matching the name of your output resource names would be sensible